Source/Dafny/Compiler-Csharp.cs

```diff
@@ -1716,32 +1716,30 @@ protected class ClassWriter : IClassWriter
       }
 
       // --- Before
-      if (bvType.NativeType == null) {
-        wr.Write("((");
-      } else {
+      if (bvType.NativeType != null) {
         if (surroundByUnchecked) {
           // Unfortunately, the following will apply "unchecked" to all subexpressions as well.  There
           // shouldn't ever be any problem with this, but stylistically it would have been nice to have
           // applied the "unchecked" only to the actual operation that may overflow.
-          wr.Write("unchecked(");
+          wr = wr.Write("unchecked").AppendChildInParenthesis();
```

Comment:
Since `AppendChildInParenthesis` doesn't take an argument, it feels more like `Fork`. Would a better name for it be `ForkInParentheses`?

Reply:
Sounds good. I've applied your suggestion.